### PR TITLE
PR for "Workshop 3" by Poonam Yadav

### PR DESCRIPTION
**Pull request by poonamyadav252**

### CODE REVIEW

File: ui/actions/nuxeo-favorites-toggle-button.js [hunk 1]
```diff
@@ -146,6 +146,9 @@ import '../nuxeo-button-styles.js';
      */
     _toggle() {
       if (!this.favorite) {
+        if (!window.confirm(this.i18n('favoritesToggleButton.confirm.add'))) {
+          return false;
```
Comment (rakeshkumar1019) [inline ui/actions/nuxeo-favorites-toggle-button.js:150]: is _toggle() function expecting bool type in return ?

File: ui/actions/nuxeo-favorites-toggle-button.js [hunk 2]
```diff
@@ -157,6 +160,9 @@ import '../nuxeo-button-styles.js';
           this._setFavorite(true);
         });
       } else {
+        if (!window.confirm(this.i18n('favoritesToggleButton.confirm.remove'))) {
+          return false;
```
Comment (rakeshkumar1019) [inline ui/actions/nuxeo-favorites-toggle-button.js:164]: same here also

**Review by rakeshkumar1019 — COMMENTED** [review]
looks good

**Comment by semisse** [comment]
@Diksha-Verma-Dev this is a test to see if Diksha gets notified on slack.

**Pull request closed without merging**